cmd/swarm/config.go

```diff
@@ -342,14 +338,10 @@ func dumpConfig(ctx *cli.Context) error {
 
 //deprecated flags checked here
 func checkDeprecated(ctx *cli.Context) {
```

Comment:
Maybe remove this altogether?

Reply:
i thought about it but also realised we would probably again deprecate stuff so I thought to just leave the infrastructure just in case...

Reply:
agree. removing